### ISSUE TITLE
lightningd crashes if plugins/README.md executable

### ISSUE THREAD

**Issue by gallizoltan** [ElementsProject/lightning#2253]
### Issue and Steps to Reproduce
I use an NTFS file system which does not handle linux permissions: means every file is executable.
(Yes, I know it is strange and dangerous and may a rare use-case. I do it for Windows compatibility.)
I've upgraded to c-lightning v0.6.3. When I started `lightningd` it crashed immediately, saying:
```
plugins/README.md: 3: plugins/README.md: Anything: not found
plugins/README.md: 5: plugins/README.md: Syntax error: "(" unexpected
Aborted (core dumped)
```
### My solution
I've just removed plugins/README.md locally. Now `lightningd` runs fine.

**Comment by rustyrussell** [comment]
Woah!  OK, not expected... @cdecker should be mask that one out explicitly?

**Comment by conscott** [comment]
Made a fix in #2260 but long term this can continue to happen with NTFS if any other non-executable files are in any plugin dir, not sure if it's worth making a note of that somewhere.

**Comment by cdecker** [comment]
No, I think we should not be setting a `--plugin-dir` by default.
